### PR TITLE
Add dcos_user custom resource

### PR DESCRIPTION
**Pull request by rbanffy**
* Add a dcos_user custom resource
* Other small changes

### CODE REVIEW

**Comment by luisdavim** [comment]
Looks like `rubocop` and `cookstyle` are conflicting with each other.

`Rubocop` says that `%w-literals should be delimited by [ and ]` and `CookStyle` is saying `%w-literals should be delimited by ( and )`....

**Comment by jjasghar** [comment]
Can you add this to the README also? I think i know what this means, but It would be nice to have the documentation around it too.

File: resources/dcos_user.rb [hunk 1]
```diff
@@ -0,0 +1,51 @@
+#
+# Cookbook Name:: dcos
+# Recipe:: default
+#
+# Copyright 2016, Chef Software, Inc.
```
Comment (jjasghar) [inline resources/dcos_user.rb]: Copyright needs to be updated here.

Comment (rbanffy) [reply resources/dcos_user.rb]: Done. The assignment, I assume, is fine.

**Comment by rbanffy** [comment]
@jjasghar we on our side think that, after this last commit, we can be happy with the results for a while. Is it good for merging now?

**Review by luisdavim — APPROVED** [review]

**Comment by wolf31o2** [comment]
I'm going to go ahead and merge this to put into `1.2.0` which I am about to release.

**Review by wolf31o2 — APPROVED** [review]
The resource does essentially the same thing as `dcos_add_user.py` shipped with DC/OS.

**Pull request merged**